setup.py

```diff
@@ -25,6 +27,17 @@
 if py_version >= (3, 5):
     packages.append('websockets/py35')
 
+ext_modules = []
+if (platform.python_implementation() == 'CPython' and
+        os.environ.get('WEBSOCKETS_CEXT_BUILD') != '0'):
```

Comment:
The 2 environment variables were there to be sure to know what we are testing and report 100% python coverage only if we are sure of what we're doing (because without cext, we get 100% coverage report). That being said, I worked on that part before adding specific tests for the cext that should also get 100% coverage so those can probably be removed.